docs/src/main/sphinx/release/release-407.md

```diff
@@ -0,0 +1,102 @@
+  columns, and queries that process tables with large Parquet row groups or Orc
+  stripes. ({issue}`15579`)
+* Improve performance of queries with window functions. ({issue}`15994`)
+* Change the `table_type` of materialized views in `information_schema.tables`
+  from `BASE TABLE` to `MATERIALIZED VIEW`. ({issue}`15350`)
+* Fix precision loss when converting `time` values with a precision higher than
+  three and `time with time zone` values with lower precision. ({issue}`15861`)
+* Fix handling of `UNKNOWN` type for row type in exchange operator. ({issue}`15948`)
+* Fix potential incorrect results when a query fails but reports normal
+  completion. ({issue}`15917`)
+
+## Delta Lake connector
+
+* Add support for the `unregister_table` procedure. ({issue}`15784`)
+* Add support for inserting into tables that have `CHECK` constraints. ({issue}`15396`)
+* Add support for writing to the [change data feed](https://docs.delta.io/2.0.0/delta-change-data-feed.html). ({issue}`15453`)
+* Improve performance of reading decimal types from Parquet files. ({issue}`15713`)
+* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Improve performance of reading string types from Parquet files. ({issue}`15897`, {issue}`15923`)
+* Improve performance of reading timestamp and boolean types from Parquet files. ({issue}`15954`)
+* Improve query performance on tables created by Trino with `CREATE TABLE AS`. ({issue}`15878`)
+* Remove support for the legacy, non-optimized Parquet writer. ({issue}`15436`)
+* Fix query failure when reading Parquet files written by Impala. ({issue}`15942`)
+* Fix listing relations failure when a Glue table has no table type set. ({issue}`15909`)
+
+## Hive connector
+
+* Reduce query latency. ({issue}`15811`)
+* Improve performance of reading decimal types from Parquet files. ({issue}`15713`)
+* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Improve performance of reading string types from Parquet files. ({issue}`15897`, {issue}`15923`)
+* Improve performance of reading timestamp and boolean types from Parquet files. ({issue}`15954`)
+* Fix query failure when reading Parquet files written by Impala. ({issue}`15942`)
+* Fix listing relations failure when a Glue table has no table type set. ({issue}`15909`)
+
+## Hudi connector
+
+* Improve performance of reading decimal types from Parquet files. ({issue}`15713`)
+* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Improve performance of reading string types from Parquet files. ({issue}`15897`, {issue}`15923`)
+* Improve performance of reading timestamp and boolean types from Parquet files. ({issue}`15954`)
+* Fix query failure when reading Parquet files written by Impala. ({issue}`15942`)
+
+## Iceberg connector
+
+* Add support for the `unregister_table` procedure. ({issue}`15784`)
+* Add the `iceberg.jdbc-catalog.connection-user` and
```

Comment:
What does "adding xxx property to the JDBC catalog"? What's the functionality being added? Should this be "Add support for specifying a user and password when connecting to the JDBC catalog"?

cc @ebyhr